examples/counter/Counter/Counter.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { shallow } from 'enzyme';
```

Comment:
Do we really need `enzyme` here? If I recall correctly there is no support in it for React 16 yet, may slow us down for the upgrade.

Reply:
There is a React 16 support in Enzyme. It might need an adapter as described in the new documentation. I'll investigate this.